src/scripts/weather/WeatherType.ts

```diff
@@ -0,0 +1,14 @@
+enum WeatherType {
+    Clear = 0,
+    Cloudy,
+    Rain,
+    Thunderstorm,
+    Snow,
+    Hail,
+    Blizzard,
+    Sunlight,
+    Sandstorm,
+    Fog,
+    Shadow,
+    Winds,
```

Comment:
I think we can safely remove `Shadow` as it's only in Pokémon XD,
And maybe change the following:
`Cloudy` → `Overcast`?
`Sunlight` → `Sunny`?
`Winds` → `Windy`?